src/config/config.ts

```diff
@@ -4,18 +4,45 @@ import { configSchema } from './config-schema';
 const safeParseConfig = (config: unknown) => {
   const result = configSchema.safeParse(config);
   if (!result.success) {
-    console.error('Invalid configuration:', result.error.format());
-    // improve error visibility in different environments
-    for (const issue of result.error.issues) {
+    console.error('Invalid configuration:');
+    result.error.issues.forEach(issue => {
       console.error(` - ${issue.path.join('.')} : ${issue.message}`);
-    }
+    });
     throw new Error('Invalid configuration');
   }
   return result.data;
 };
-const getConfig = () => {
-  return safeParseConfig(configIn);
+
+const appConfig = configIn.get('app') as Record<string, unknown> | undefined;
+const databaseConfig = configIn.get('database') as
+  | Record<string, unknown>
+  | undefined;
+const jwtConfig = configIn.get('JWT') as Record<string, unknown> | undefined;
+
+const nodeEnvFromConfig =
```

Comment:
Why are you adding more configuration here? This doesn't make any sense. Please remove this configuration.
The base configuration is agnostic, so if you need a new setting, you only need to add it to.
config/default.json
config/custom-environment-variables.json
in the repository.